Recording doesn't work

I'm using retroarch flatpak with default settings and when I choose "Start Recording", the menu closes, but nothing happens.

It seems to be due to libx264 not being found by ffmpeg. With the default settings, the logs say:

[ERROR] [FFmpeg]: Cannot find vcodec libx264.

Changing the codec to WebM from main settings works.

--- a/retroarch.cfg
+++ b/retroarch.cfg
@@ -42,3 +42,6 @@ overlay_directory = "@prefix@/share/libretro/overlays"
 
 # If disabled, will hide 'Online Updater' inside the menu.
 menu_show_core_updater = "true"
+
+# Use WebM for video recording
+video_record_quality = "6"

